docs/source/model_design/risk_factors.rst

```diff
@@ -125,6 +126,132 @@ Mediation
+
+
+Risk exposures in GBD
+---------------------
+
+GBD estimates always pertain to the mid-year or yearly average measurements of 
+a population with a specific location, year, sex, and age, or an aggregation of 
+some such populations. Thus, in the context of GBD, a risk exposure is a 
+*distribution of individual exposure values* within a location-year-sex-age-
+population.
+
+If the exposure is dichotomous, for each location, sex, and year, GBD 
+will estimate a continuous age trend of the proportion of, say, individuals with
+BMI over 30. If the exposure is continuous, then GBD estimates the distribution of the 
+exposure variable over the population in each age, sex, year, and location.
+
+GBD's risk exposures will generally be less reliable than GBD cause of death 
+models, and when designing a risk exposure, it is important to both learn from
+the GBD modeler what the entity captured by their exposure model is.
+
+	Take, for example, the GBD exposure *has ever experienced 
+	intimate partner violence*. Barring incredibly high mortality rates among 
+	IPV victims, we would expect the proportion of the population that has ever 
+	experienced IPV to increase monotonically with age. However, survey data 
+	consistenly reports this proportion to peak among 30-40 year olds, which is 
+	refleced in the GBD model. We believe this phenomenon to be the result of 
+	recall bias. When implementing this in a model, however, if we were to 
+	initialize a population with dichotomous and persistent IPV exposure values 
+	from GBD estimates and then allow the simulants to age for 10 years, our 
+	exposure distribution would no longer match our reference data. Thus it 
+	becomes clear that the entity we're describing needs to be "recollection of 
+	IPV", "recent experience of IPV", or some other attribute that incorporates a 
+	time component.
+
+Risk exposures in Vivarium
+--------------------------
+
+In Vivarium, each simulant will be assigned an exposure value. We will 
+typically derive these values from a population-level distribution provided by a 
+GBD risk exposure.
+
+Any given attribute that we are interested in may be codified in a variety of 
+ways. The choices to make include which distribution to use, how to measure the 
+risk, and what time frame within which to consider the risk. We include some 
+examples below.
+
+	Say we are modeling *BMI* as a risk exposure. BMI could be 
+	included as a continuous variable, or binned into {<20, 20-25,>25}. This 
+	decision will be based on the outcomes of interest and data availability.
+
+	If we are interested in BMI as a risk for IHD, we might only be interested
+	in current BMI. However, if we are modeling BMI as a risk for osteoporosis,
+	it is possible that we will be interested in the cumulative history of 
+	BMI.
+
+	Assume we are intersted in capturing *smoking* as a risk exposure. If the 
+	outcome of interest is pneumoconiosis, we will be interested in a subject's 
```

Comment:
/pneumoconiosis/lung cancer/ --- on the grounds that it can be overwhelming to a reader to switch between examples so we should reduce the variety of applications when possible.